templates/commands/render/render_action.go

```diff
@@ -146,19 +147,19 @@ func templateFuncs() template.FuncMap {
 		"contains":          strings.Contains,
 		"replace":           strings.Replace,
 		"replaceAll":        strings.ReplaceAll,
-		"sortStrings":       sortStrings,
+		"sortStrings":       utils.SortStrings,
 		"split":             strings.Split,
 		"toLower":           strings.ToLower,
 		"toUpper":           strings.ToUpper,
 		"trimPrefix":        strings.TrimPrefix,
 		"trimSuffix":        strings.TrimSuffix,
 		"trimSpace":         strings.TrimSpace,
-		"toSnakeCase":       toSnakeCase,
-		"toLowerSnakeCase":  toLowerSnakeCase,
-		"toUpperSnakeCase":  toUpperSnakeCase,
-		"toHyphenCase":      toHyphenCase,
-		"toLowerHyphenCase": toLowerHyphenCase,
-		"toUpperHyphenCase": toUpperHyphenCase,
+		"toSnakeCase":       utils.ToSnakeCase,
```

Comment:
Could we leave these in the "render" package? Is there an important reason to move them into utils?

Reply:
Never mind, I understand now.